commune_app/models.py

```diff
@@ -1,3 +1,51 @@
-# from django.db import models
+from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import AbstractUser
+from django.core.validators import RegexValidator
 
-# Create your models here.
+# regex to filter valid phone numbers
+phone_regex = RegexValidator(regex=r'^\+?1?\d{9,15}$', message="Phone number format: '+999999999'")
+
+
+class User(AbstractUser):
```

Comment:
This code already merged as part of:
https://github.com/redhat-beyond/commune/pull/27

please rebase